CONTRIBUTING.md

```diff
@@ -48,6 +48,27 @@ There are currently no automatic tests done after a Pull Request.
 
 We will then do our best to answer you and test your contribution as soon as possible, before merging it. We may iterate with you to converge before merging to master branch.
 
+### Backports
+
+For backports, we use a workflow similar to the [Ansible Development
+Cycle](https://docs.ansible.com/ansible/latest/community/development_process.html#backporting-merged-prs).
```

Comment:
Ansible project does not have any `master` branch indeed, but a `devel` branch which is the default branch of the project, used to push new code/feat, and `stable-X.Y` branches for maintenance. Our workflow is similar to the Ansible one. The name of the branch could be `devel`, `master` or even `unstable`, it would not change the workflow:

PR to the default branch -> wait for merge -> backport to maintenance branch.

I thought it was not confusing in the proposed example.

- In this PR example: `git cherry-pick -x <commit_from_master>`
- In Ansible's example: `git cherry-pick -x [SHA_FROM_DEVEL]`

However, to avoid any misunderstanding, I will add a note to explain our default branch is named `master`, not `devel`. (I don't care to rename `master` to `devel` though, we can discuss this in the chat).